src/tests/topp/OpenSwathWorkflow_11_output.featureXML

```diff
@@ -12,7 +12,7 @@
 			<intensity>141615</intensity>
 			<quality dim="0">0</quality>
 			<quality dim="1">0</quality>
-			<overallquality>-1.58727</overallquality>
```

Comment:
Several scores differ with SONAR parameters enabled. Something is not working right yet.